junk-drawer/timers.janet

```diff
@@ -1,4 +1,4 @@
-(use ./ecs)
```

Comment:
This change goes against the stated move to `use`. May I ask what the reasons are? I am asking out of curiosity.

Reply:
huh, I  thought I changed these back to `use` :facepalm: , must have forgot to commit it. I'll blame git

Reply:
imports back to `use`